docs/contributing.rst

```diff
@@ -89,10 +89,12 @@ Releasing the theme
 
 To release a new version of the theme, core team will take the following steps:
 
-#. Bump the version in ``sphinx_rtd_theme/__init__.py``, ``setup.py`` and
-   ``package.json``.  We follow `semver <http://semver.org/>`_ and `PEP440`_
+#. Bump the version by running ``bump2version [major|minor|patch|dev]``.
```

Comment:
Minor nitpick: I'm a little confused by how to call this. The way I parse this is if I want to bump the minor release number, I'd run: `bump2version minor` and we'd have `0.5.0dev0`.

It's not supposed to be run as `bump2version 0.5.0dev0` is it?

Reply:
Correct.
We are at `0.4.3.dev0` now, if you run `bump2version dev`, the version will be `0.4.3.dev1`.
If you run `bump2version minor`, the version will be `0.5.0.dev0`.

If you want to force a particular version, you can run `bump2version --new-version 0.9.2 patch` (the word `patch` has no effect here).

I've added some more text to the instructions